spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/convert/TwoStepsConversions.java

```diff
@@ -92,6 +94,11 @@ public TwoStepsConversions(CustomConversions customConversions, ObjectToKeyFacto
 
 	}
 
+	@Override
+	public GenericConversionService getConversionService() {
```

Comment:
I don't think it's a good idea to expose conversion service, since we actually need two-steps conversion on read. convertOnReadSingle should be used for that.